package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "fastify-formbody",
-  "version": "3.1.0",
+  "version": "3.2.0",
```

Comment:
You should not be updating the version number. That is something the package maintainers should be doing.